api/api.yaml

```diff
@@ -0,0 +1,42 @@
+    openapi: 3.0.0
+    info:
+      title: TranslateApp API
+      description: Api in Golang.
+      version: 0.0.1
+    servers:
+      - url: http://localhost:8080/
+        description: Optional server description, e.g. Main (production) server
+    paths:
+      /languages:
+        get:
+          summary: Returns a list of languages.
+          description: You can take one language from the list as option in translating word.
+          responses:
```

Comment:
Status 200 is for success only. I think we should also add 500 here.
Please also take a look at guides from Microsoft and Google how to deal with errors and how to return information on errors.